config/puma.rb

```diff
@@ -0,0 +1,33 @@
+# Adjust setting below to your local environment and than start puma like this:
+# puma -C config/puma.rb --control=unix:///tmp/puma.ctrl --control-token=
+# and manage it like this:
+# pumactl --control-url=unix:///tmp/puma.ctrl {restart|stop|status|stats|halt}
+# For more information see http://www.rubydoc.info/gems/puma/
+
+environment 'production'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.